### PR TITLE
Update Studies "Learn More" link to point to Focus Studies support page

### PR DESCRIPTION
**Pull request by amedyne**
For #6104 

Update Studies "Learn More" link to point to Focus Studies support page.

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt [hunk 1]
```diff
@@ -49,7 +49,7 @@ object SupportUtils {
         WHATS_NEW_KLAR("whats-new-firefox-klar-android"),
         SEARCH_SUGGESTIONS("search-suggestions-focus-android"),
         ALLOWLIST("focus-android-allowlist"),
-        STUDIES("how-opt-out-studies-firefox-android")
+        STUDIES("how-opt-out-studies-firefox-focus-android")
```
Comment (ionutbedregeanu) [inline app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt:52]: It seems that this page doesn't exist

Comment (mcarare) [reply app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt:52]: @kelimuttu Do you know what would be the correct link for this or could you point me to someone else who might know? Thank you!

P.S.: I searched and I could not find a topic on this for Focus Android. One for iOS does exist (https://support.mozilla.org/en-US/kb/studies-focus-ios).

Comment (kelimuttu) [reply app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt:52]: Hi @mcarare I checked and confirmed that we don't have one for Focus Android. But I just created [this one](https://support.mozilla.org/en-US/kb/studies-focus-android) for placeholder. Feel free to use that link and I'll tell our content manager to work on the article later. 

Will that work for you?

Comment (mcarare) [reply app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt:52]: Great! Thank you!

Comment (kelimuttu) [reply app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt:52]: Sure thing

**Comment by ionutbedregeanu** [comment]
I'm closing this pr in favour of https://github.com/mozilla-mobile/focus-android/pull/6116

**Pull request closed without merging**